ServeEasyApp/app.py

```diff
@@ -0,0 +1,170 @@
+from flask import Flask,render_template,request,redirect
+from flask_mysqldb import MySQL
+import MySQLdb
+
+
+app=Flask(__name__)
+
+import yaml
+db = yaml.load(open('db.yaml'))
+app.config['MYSQ_HOST'] = db['mysql_host']
+app.config['MYSQL_USER'] = db['mysql_user']
+app.config['MYSQL_PASSWORD'] = db['mysql_password']
+app.config['MYSQL_DB'] = db['mysql_db'] 
+mysql = MySQL(app)
+
+
+user_details=()
+@app.route('/home')
+def home():
+    return render_template('home.html',user_details=())
+
+@app.route('/index2')
+def index2():
+    return render_template('index2.html')
+
+@app.route('/home/<int:user_id>')
+def user_home(user_id):
+    try:
+        cursor = mysql.connection.cursor(MySQLdb.cursors.DictCursor)
+        query="SELECT name,username,email,phone,user_id from user where user_id = "+str(user_id)
+        cursor.execute(query)
+        user_details = cursor.fetchall()
+        return render_template('home.html',user_details=user_details)
+    except Exception as e:
+        return str(e)
+
+
+@app.route('/sign_up',methods=['POST','GET'])
+def sign_up():
+    if(request.method=='POST'):
+        user_details = request.form
+        name = user_details['name']
+        username = user_details['username']
+        email = user_details['email']
+        password = user_details['password']
+        phone = user_details['phone']
+        
+        cur = mysql.connection.cursor()
+        cur.execute("insert into user value(%s,%s,%s,%s,%s)",(name,0,username,email,phone))
+        mysql.connection.commit()
+        cur.close()
+        
+        return redirect('/home')
+    return render_template('sign_up.html')
+
+@app.route('/sign_in',methods=['POST','GET'])
+def sign_in():
+    if(request.method=='POST'):
+        user_details = request.form
+        username = user_details['username']
+        user_password = user_details['password']
+        # print(username,password)
```

Comment:
- [ ] Don't comment such lines, remove after debugging
- [ ] If any portion is written for debug then add comments like this part is for debuging